spring-cloud-gcp-starters/spring-cloud-gcp-starter-trace/README.adoc

```diff
@@ -0,0 +1,60 @@
+= Google Cloud Stackdriver Trace Spring Boot Starter
+
+Maven coordinates, using Spring Cloud GCP BOM:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-gcp-starter-trace</artifactId>
+</dependency>
+----
+
+Gradle coordinates:
+
+[source]
+----
+dependencies {
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-gcp-starter-trace', version: '1.0.0.BUILD-SNAPSHOT'
+}
+----
+
+This starter uses Spring Cloud Sleuth and provides a `SpanReporter` that sends the Sleuth's trace information to Stackdriver Trace.
+
+All configurations are optional. You can override a number of default configuration values in addition to the core
+`spring.cloud.gcp.project-id` and `spring.cloud.gcp.credentials-location`.
+
+By default, this starter will send trace asynchronously using a flushable trace consumer that auto-flushes when
+its buffered trace messages exceed its buffer size or have been buffered for longer than its scheduled delay.
+
+The following properties are optional:
+
+[source,yaml]
+----
+# Number of threads to use by the underlying gRPC channel to send the trace request to Stackdriver
+spring.cloud.gcp.trace.executor-threads=[NUMBER_OF_SENDER_THREADS]
+
+# Buffer size, defaults to 1% of Runtime.totalMemory(), traces will be flushed to Stackdriver when buffered trace
+# messages exceed this size.
+spring.cloud.gcp.trace.buffer-size-bytes=[BUFFER_SIZE_BYTES]
+
+# Buffered trace messages will be flushed to Stackdriver when buffered longer than scheduled delays.
+spring.cloud.gcp.trace.scheduled-delay-seconds=[SCHEDULED_DELAY_SECONDS]
+----
+
+You can use core Spring Cloud Sleuth properties to control Sleuth's sampling rate, etc. For example:
+
+[source,yaml]
+---
+spring.sleuth.sampler.percentage=.1                             # Send 10% of the request traces to Stackdriver
```

Comment:
What is the default? When testing this out, the user may be confused why the traces don't show up in Stackdriver if the default is anything but `1.0`. Can we draw a bit more attention to this in the readme?

Reply:
This is already called out in Sleuth's doc.
https://github.com/spring-cloud/spring-cloud-sleuth#features

We should probably just link it.